components/new-homepage/TheFutureIsCollectiveSection.js

```diff
@@ -0,0 +1,129 @@
+                id="home.futureIsCollective.description"
+                defaultMessage={
+                  'Get access to money with nonprofit, co-op, or business status now. Join the movement for budget transparency and community-control of technology.'
+                }
+              />
+            </P>
+          </Box>
+          <Box mb="40px" display="flex" flexDirection={['column', 'row']} alignItems="center">
+            <Link href="/create">
+              <StyledButton
+                width={['288px', '125px']}
+                my={['12px', null, 0]}
+                mr={[0, '24px']}
+                buttonStyle="marketing"
+                whiteSpace="nowrap"
+                backgroundColor="primary.900"
+                fontSize="16px"
+                lineHeight="20px"
+              >
+                <FormattedMessage id="home.startNow" defaultMessage="Start Now" />
+              </StyledButton>
+            </Link>
+            <StyledButton
+              onClick={() => setShowModal(true)}
+              my={['12px', null, 0]}
+              width={['288px', '196px']}
+              fontSize="16px"
+              lineHeight="20px" // TODO Fix border color and text decoration
+            >
+              <Span mr={'14px'} textDecoration="underline">
+                <FormattedMessage id="home.makeCommunitySection.watchVideo" defaultMessage="Watch Video" />
+              </Span>
+              <Span className="arrowIcon">
+                <RightArrow size="15" />
+              </Span>
+            </StyledButton>
+          </Box>
+          <Box display={[null, 'none']}>
+            <NextIllustration
+              width={320}
+              height={589}
+              alt="The future is collective."
+              src="/static/images/new-home/budget-illustration-xs.png"
+            />
+          </Box>
+          <Box display={['none', 'block', 'none']}>
+            <NextIllustration
+              width={768}
+              height={431}
+              alt="The future is collective."
+              src="/static/images/new-home/budget-illustration-sm.png"
+            />
+          </Box>
+          <Box display={['none', null, 'block']}>
+            <NextIllustration
+              width={956}
+              height={552}
+              alt="The future is collective."
+              src="/static/images/new-home/budget-illustration-md.png"
+            />
+          </Box>
+        </Flex>
+      </Flex>
+      {showModal && (
+        <StyledModal
+          padding="0"
+          background="transparent"
+          width={[1, null, '670px', null, '770px']}
+          onClose={() => setShowModal(false)}
+        >
+          <Container display="flex" width={1} height={400} maxWidth={712} background="black">
+            <iframe
+              title="YouTube video"
+              width="100%"
+              height="400px"
+              src="https://www.youtube.com/embed/IBU5fSILAe8"
```

Comment:
Small privacy enhancement

```suggestion
              src="https://www.youtube-nocookie.com/embed/IBU5fSILAe8"
```